material-lists/index.ets

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022 Application Library Engineering Group.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export {SingleLineListItems , TwoLineListItems,ThreeLineListItems,NestedListItems} from './src/main/ets/components/MainPage/listUtilities/list_classes'
+export{SingleLineListModel,TwoLineListModel ,ThreeLineListModel} from './src/main/ets/components/MainPage/listModels/listmodels'
```

Comment:
format the code and remove empty lines

Reply:
formatted  the code and deleted the unnecessary lines